control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources.go

```diff
@@ -1039,6 +1045,18 @@ func (r *reconciler) reconcileCloudConfig(ctx context.Context, hcp *hyperv1.Host
 	return nil
 }
 
+func (r *reconciler) reconcileGuestClusterAlertRules(ctx context.Context) error {
```

Comment:
what if I wanted to opt out the alert or I want to change the config?

Reply:
@csrwng confirmed that we need to reconcile the rules, please see https://coreos.slack.com/archives/C01C8502FMM/p1652819843662269?thread_ts=1648143232.433299&cid=C01C8502FMM

Reply:
I question even these metrics should be scrapped by anything in the guest cluster, we'll probably want to make all this opt-in/opt-out, no need to solve in this PR though.

Reply:
In standalone OCP these are not optional, they are constantly reconciled by the cluster-kube-apiserver-operator. I don't think we should make them optional in hypershift clusters.

Reply:
correct, it's about their workloads using deprecated APIs